mongodb.go

```diff
@@ -7,33 +7,34 @@ import (
 	"time"
 )
 
+const mongoDBPort = 27017
+
 // SetupMongoContainer sets up a real MongoDB instance for testing purposes,
 // using a Docker container. It returns the container ID and its IP address,
 // or makes the test fail on error.
 func SetupMongoContainer() (c ContainerID, ip string, port int, err error) {
 	port = RandomPort()
-	forward := fmt.Sprintf("%d:%d", port, 27017)
+	forward := fmt.Sprintf("%d:%d", port, mongoDBPort)
 	if BindDockerToLocalhost != "" {
 		forward = "127.0.0.1:" + forward
 	}
-	c, ip, err = SetupContainer(MongoDBImageName, port, 10*time.Second, func() (string, error) {
-		res, err := run("--name", GenerateContainerID(), "-d", "-P", "-p", forward, MongoDBImageName)
-		return res, err
+	c, ip, err = SetupContainer(MongoDBImageName, mongoDBPort, 10*time.Second, func() (string, error) {
```

Comment:
I'm not sure that replacing the random "outside" port with the static "inside" port will work. What was your intention?